components/org.wso2.carbon.identity.cors.valve/src/main/java/org/wso2/carbon/identity/cors/valve/constant/RequestMethod.java

```diff
@@ -23,6 +23,21 @@
  */
 public class RequestMethod {
 
+    /**
+     * "GET" method.
+     */
```

Comment:
Do we need multiline comments for these?

Reply:
Removed